### PR TITLE
Add basic integration test

### PR DESCRIPTION
**Pull request by ethax-ross**
We will be end-to-end testing separately to ensure the integration with Dynamics is functioning correctly, however we won't run these tests as often as the unit/integration tests. This PR adds a basic integration test that will test the full request/response stack.

### CODE REVIEW

File: GetIntoTeachingApi/Models/MappingInfo.cs [hunk 1]
```diff
@@ -13,6 +13,10 @@ public class MappingInfo
         public IDictionary<string, IDictionary<string, string>> Relationships { get; set; } =
             new Dictionary<string, IDictionary<string, string>>();
 
+        public MappingInfo()
```
Comment (DrLazer) [inline GetIntoTeachingApi/Models/MappingInfo.cs:16]: Is this needed?

Comment (ethax-ross) [reply GetIntoTeachingApi/Models/MappingInfo.cs:16]: Yeah you need a parameterless constructor to be able to deserialise into the object

File: GetIntoTeachingApi/Utils/Env.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@ public class Env : IEnv
         public bool IsDevelopment => EnvironmentName == "Development";
         public bool IsProduction => EnvironmentName == "Production";
         public bool IsStaging => EnvironmentName == "Staging";
+        public bool IsTest => EnvironmentName == null;
```
Comment (DrLazer) [inline GetIntoTeachingApi/Utils/Env.cs:10]: is that a string check for null?  should it be string.IsNullOrEmpty() perhaps?

Comment (ethax-ross) [reply GetIntoTeachingApi/Utils/Env.cs:10]: It'll never be an empty string in this case; there's no really nice way of checking if we're running in the test environment vs dev, but we explicitly set the env name for all other environments so the `null` check works here

File: GetIntoTeachingApi/Models/MappingInfo.cs (hunk 1 above)
Comment (DrLazer) [reply GetIntoTeachingApi/Models/MappingInfo.cs:16]: ok cool

File: GetIntoTeachingApi/Utils/Env.cs (hunk 2 above)
Comment (DrLazer) [reply GetIntoTeachingApi/Utils/Env.cs:10]: cool

**Pull request merged**